packages/terra-clinical-data-grid/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## Unreleased
+* Fixed
+  * Fixed section id to include datagrid id for supporting multiple grids on the same page.
```

Comment:
Can we rephrase this so that it's more consumer friendly?
```suggestion
* Added
  * Added tabbing support for multiple grids on the same page.
```